packages/graphql-codegen-cli/src/yml.ts

```diff
@@ -1,7 +1,19 @@
 import { Types, debugLog } from 'graphql-codegen-core';
 import { safeLoad } from 'js-yaml';
 
+export function interpolate<T extends { [key: string]: string }>(str: string, context: T) {
+  // tslint:disable-next-line:forin
+  for (let key in context) {
```

Comment:
Maybe we should replace all variables outside of `context` with `undefined`?

Something like: `/($([a-z0-9\-_]))/g` and replace `$ASD` with `process.env.ASD` which will give an undefined if value is not available.

Reply:
Now we iterate through all properties of `process.env` and if there's a variable that is not a part of that we end up with a string, `$ASD` for example.

Reply:
Done :)

Reply:
I don't trust `.*` but if it works (as test shows) then I'm okay with it :))